src/ts/components/Suggestion.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useContext } from "react";
-import jsDiff, { Change } from "diff";
+import { Change, diffChars } from "diff";
```

Comment:
Mixing default and named exports in this way doesn't make too much sense given the usage below, and `jsDiff` doesn't have a default export (upsetting vite).